libraries/AP_Networking/AP_Networking_Config.h

```diff
@@ -43,3 +60,13 @@
 #ifndef AP_NETWORKING_DEFAULT_MAC_ADDR
 #define AP_NETWORKING_DEFAULT_MAC_ADDR "C2:AF:51:03:CF:46"
 #endif
+
+#ifndef AP_NETWORKING_TESTS_ENABLED
+#define AP_NETWORKING_TESTS_ENABLED 0
+#endif
+
+#if AP_NETWORKING_TESTS_ENABLED
+#ifndef AP_NETWORKING_TEST_IP
+#define AP_NETWORKING_TEST_IP "192.168.13.15"
```

Comment:
Make this the same value as the one in here: `AP_DDS_DEFAULT_UDP_IP_ADDR "192.168.13.2"`

Reply:
fixed